<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetDialog</class>
 <widget class="QDialog" name="DataSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно выбора набор данных</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите набор данных из списка:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hor5">
     <item>
      <widget class="QListView" name="lstDataSet">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Название набора данных:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Комментарий:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtComment"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblLabelDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Время создания:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hor6">
         <item>
          <widget class="QPushButton" name="btnChange">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNew">
           <property name="text">
            <string>Создать новый</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hor4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>DataSetDialog</receiver>
   <slot>slotOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DataSetDialog</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChange</sender>
   <signal>clicked()</signal>
   <receiver>DataSetDialog</receiver>
   <slot>slotUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>DataSetDialog</receiver>
   <slot>slotDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNew</sender>
   <signal>clicked()</signal>
   <receiver>DataSetDialog</receiver>
   <slot>slotAddNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChangeDataSet()</slot>
  <slot>slotOk()</slot>
  <slot>slotCancel()</slot>
  <slot>slotUpdate()</slot>
  <slot>slotDelete()</slot>
  <slot>slotAddNew()</slot>
 </slots>
</ui>
